pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1071,8 +1071,15 @@ func TestPodTimestamp(t *testing.T) {
 			queue.MoveAllToActiveQueue()
 		}
 	}
+	backoffPod := func(pInfo *podInfo) operation {
+		return func() {
+			queue.backoffPod(pInfo.pod)
+		}
+	}
 	flushBackoffQ := func() operation {
 		return func() {
+			// wait for pod to complete backoff.
+			time.Sleep(time.Second)
```

Comment:
A better approach is to use `NewPriorityQueueWithClock` instead of `NewPriorityQueue` at line 1122 and pass a `FakeClock` (k8s.io/apimachinery/pkg/util/clock) to it. You can then control the time with the fake clock, for example by calling its `Step` function.

Reply:
Done thank you for your help.